plugins/outputs/sql/sample.conf

```diff
@@ -29,6 +29,15 @@
   ##  {TABLE} - tablename as a quoted identifier
   # table_exists_template = "SELECT 1 FROM {TABLE} LIMIT 1"
 
+  ## Table update template, available template variables:
+  ##  {TABLE} - table name as a quoted identifier
+  ##  {COLUMNS} - column definitions (list of quoted identifiers and types)
```

Comment:
Is this really `{COLUMNS}` or `{COLUMN}`?

Reply:
It's a typo.
```suggestion
  ##  {COLUMN} - column definition (quoted identifier and type)
```